CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ A breaking change will get clearly marked in this log.
 
 ### Breaking Changes
 * **This update supports Protocol 21**. It is an additive change to the protocol so there are no true backwards incompatibilities, but your software may break if you encounter new unexpected fields from this Protocol ([#949](https://github.com/stellar/js-stellar-sdk/pull/949)).
-* **Default timeout for transaction calls is now set to 60 seconds from previous default of 10**. 10 seconds is often not enough time to review transactions before signing, which would cause a `txTooLate` error response from the server.
+* **Default timeout for transaction calls is now set to 300 seconds (5 minutes) from previous default of 10 seconds**. 10 seconds is often not enough time to review transactions before signing, especially in Freighter or using a hardware wallet like a Ledger, which would cause a `txTooLate` error response from the server. Five minutes is also the value used by the CLI, so this brings the two into alignment.
```

Comment:
This needs to go into the “Unreleased” section rather than edit the changelog

Reply:
Good with 300 seconds / 5 mins 👍

Reply:
oops, sorry about that, updated